blas/netlib/generate_blas.go

```diff
@@ -827,8 +859,21 @@ var (
 // an API that allows client calls to specify order, so this is here to document that fact.
 type order int
 
+const rowMajor order = 101
+
 const (
-	rowMajor order = 101 + iota
+	cblasNoTrans   = 111
```

Comment:
Are these hardcoded constants ok or should we get them as `C.CblasNoTrans`, etc?

Reply:
Let's use the `C.<label>` constants.

Reply:
Done.